lib/govuk_navigation_helpers/taxonomy_sidebar.rb

```diff
@@ -9,7 +9,7 @@ def initialize(content_item)
 
     def sidebar
       {
-        items: taxons
+        items: [taxons, elsewhere_on_govuk, elsewhere_on_the_web].flatten
```

Comment:
Is there any reason you use `flatten` instead of just concatenating with `+` ? Are there nested arrays or something?